### ISSUE TITLE
Google workingLocation fix part II: 2-way sync for recurring series

### ISSUE THREAD

**Issue by phw198** [phw198/OutlookGoogleCalendarSync#1843]
**OGCS Version**: v2.10.4.0  
**Sync Direction**: O<->G

**Bug Description**  
OGCS is not able to update a Google "working location" recurring event that starts before the date range being synced.

**Steps to Reproduce the Issue**  
1. Sync a working location series G->O before v2.10.4 prevented this.
1. Modify the series in Outlook somehow
2. OGCS fails to sync the modification back to Google.

**Any other information**  
:paperclip: Attach your `OGcalsync.log` file.
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/14975635/OGcalsync.log)
```
2024-04-15 09:38:14,380 DEBUG  14 OutlookGoogleCalendarSync.GoogleOgcs.Calendar [797] -  Processing >> 20.01.2023 (R) => "Zuhause"
2024-04-15 09:38:14,382 DEBUG  14 OutlookGoogleCalendarSync.Recurrence [32] -  Creating Google iCalendar definition for recurring event.
2024-04-15 09:38:14,391 DEBUG  14 OutlookGoogleCalendarSync.Recurrence [48] -  RRULE:FREQ=WEEKLY
2024-04-15 09:38:14,393 WARN  14 OutlookGoogleCalendarSync.GoogleOgcs.Calendar [1042] -  Evading Google bug - not allowing default calendar notification to be (re?)set for all-day event.
2024-04-15 09:38:14,394 DEBUG  14 OutlookGoogleCalendarSync.Console [348] -  Recurrence: RRULE:FREQ=WEEKLY;BYDAY=FR => RRULE:FREQ=WEEKLY
2024-04-15 09:38:14,395 DEBUG  14 OutlookGoogleCalendarSync.Console [348] -  Privacy: public => default
2024-04-15 09:38:14,395 DEBUG  14 OutlookGoogleCalendarSync.Console [348] -  Free/Busy: transparent => opaque
2024-04-15 09:38:14,395 DEBUG  14 OutlookGoogleCalendarSync.GoogleOgcs.Calendar [1071] -  Saving timestamp when OGCS updated event.
2024-04-15 09:38:14,595 FAIL  14 OutlookGoogleCalendarSync.GoogleOgcs.Calendar [2243] -  The service calendar has thrown an exception. HttpStatusCode is BadRequest. A working location event must have a transparency setting of transparent.
```

_Originally posted by @lestarte and @hakarune in https://github.com/phw198/OutlookGoogleCalendarSync/issues/1817#issuecomment-2055921939_

**Comment by phw198** [comment]
:package: This should now be resolved with [hotfix v2.10.4.3.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/15051894/v2.10.4.3.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; [CRCs](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Known-Issues#cyclic-redundancy-checks---aka-crcs): Zip = `2F1A77F7` Exe = `D3E9327B`

Please note, because working locations are not supported, it will mean that any that have been synced to Outlook whilst using OGCS v2.10.3 or earlier **will now be removed from Outlook**.

**Comment by lestarte** [comment]
@phw198 Thanks, Paul.
**It works** smoothly again, finally :)
And the old ones were safely removed from Outlook, too.
Damn you, Google!
Praise you, OGCS / Paul!
